bevy_lint/README.md

```diff
@@ -1,9 +1,168 @@
+<div class = "rustdoc-hidden">
+
 # `bevy_lint`
 
 `bevy_lint` is a custom linter for the [Bevy game engine](https://bevyengine.org), similar to [Clippy](https://doc.rust-lang.org/stable/clippy).
 
+</div>
+
+<div class="rustdoc-alert rustdoc-alert-warning">
+
+> **Warning**
+>
+> This is an unofficial community project, hacked upon by the Bevy CLI working group until it is eventually upstreamed into the main [Bevy Engine organization](https://github.com/bevyengine). Pardon our rough edges!
+
+</div>
+
+## Installation
+
+`bevy_lint` depends on a pinned nightly version of Rust with the `rustc-dev` Rustup component. This is because `bevy_lint` uses [internal `rustc` crates](https://doc.rust-lang.org/nightly/nightly-rustc/) that can only be imported with the permanently-unstable [`rustc_private` feature](https://doc.rust-lang.org/nightly/unstable-book/language-features/rustc-private.html). You can refer to the [compatibility table](#compatibility) to see which version of the linter requires which toolchain.
```

Comment:
Nit: might be worth calling out here or somewhere else whether or not this tool will ever work on stable

Reply:
> This is because `bevy_lint` uses [internal `rustc` crates](https://doc.rust-lang.org/nightly/nightly-rustc/) that can only be imported with the permanently-unstable [`rustc_private` feature](https://doc.rust-lang.org/nightly/unstable-book/language-features/rustc-private.html).

I do mention how `#![feature(rustc_private)]` is permanently unstable, so I think it is implied that `bevy_lint` will never work on stable. Do you think I need to be more clear about this?

Reply:
I'm going to hit the merge button, since this is non-blocking. If you have any further concerns, just let me know and I can open another PR :)